CanvasCore/CanvasCore/People/Backdrop/BackdropPickerViewController.swift

```diff
@@ -240,6 +240,8 @@ open class BackdropPickerViewController: UIViewController, UICollectionViewDeleg
     
     open func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! BackdropCell
+        cell.isAccessibilityElement = true
+        cell.accessibilityLabel = String(format: NSLocalizedString("Image %d", comment: "Image # in collection"), indexPath.item)
```

Comment:
I think we need to be specifying the bundle for `NSLocalizedString`.

```
NSLocalizedString("Image %d", bundle: .core, value: "", comment: "")
```

`.core` in CanvasCore does reference the correct `CanvasCore` bundle. 😆 so many cores.

Reply:
Good call